tests/test_glm_runs.py

```diff
@@ -58,4 +57,5 @@ def test_setup_Mbasis():
     model.fit(spike_data)
     model.predict(spike_data)
     key, subkey = jax.random.split(key)
-    X = model.simulate(subkey, 20, spike_data[:, :100])
\ No newline at end of file
+    X = model.simulate(subkey, 20, spike_data[:, :100])
+    return
```

Comment:
is return necessary for tests?

Reply:
nope, I'll remove it (but generally I like a return statement, it makes it clear when the function ends, especially if your last line of code is more indented then the function/method body, like if you have some if statement)